### DISCUSSION TITLE
Remap command with parameters

### DISCUSSION THREAD

**Discussion by CptPotato** [helix-editor/helix#5963] [Q&A] [answered]
Is it possible to map commands that require additional inputs to a key? I'm thinking of something along these lines:

```toml
a = ["find_next_char x"]
```

I feel like there was an issue related to this, but wasn't able to find one.

**Comment by ath3** [comment, marked as answer]
I think this needs pr #4709

**Reply by CptPotato** [reply]
Thanks, this is exactly what I was looking for.
